### DISCUSSION TITLE
Evil twin AP

### DISCUSSION THREAD

**Discussion by minanagehsalalma** [risinek/esp32-wifi-penetration-tool#3] [Ideas]

**Comment by risinek** [comment]
It's not yet implemented as a standalone functionality, but it basically is included in deauthentication attack using rogue AP. 
The rogue AP is evil twin as it clones original AP, and can be easily started on different channel (currently it's started on same channel as the genuine AP).

I'm gonna update the repo with some implementation proposal.

**Reply by minanagehsalalma** [reply]
> I'm gonna update the repo with some implementation proposal.

Ok 

> but it basically is included in deauthentication attack using rogue AP.
The rogue AP is evil twin as it clones original AP

I know , but can both of them be used at the same time ? Deauth and rogue ap?
